### DISCUSSION TITLE
Problem while deploying project

### DISCUSSION THREAD

**Discussion by theliuk** [mia-platform/community#69] [Q&A] [answered]
Hi, 
I'm deploying a project for the first time in the production environment using MLP and the pipeline returns the following error:

```
$ mlp interpolate -e "${ENVIRONMENT_PREFIX}" -e "${ENVIRONMENT_VARIABLES_PREFIX}" -f "${SOURCE_PATH}"/"${ENVIRONMENT_TO_DEPLOY}" -f "${SOURCE_PATH}" -o "${DESTINATION_PATH}"
$ mlp deploy --ensure-namespace=false --server "${KUBE_URL}" --certificate-authority /tmp/kube-ca.pem --token "${KUBE_TOKEN}" --deploy-type "${DEPLOY_TYPE}" --force-deploy-when-no-semver=${FORCE_DEPLOY_WHEN_NO_SEMVER} -f ${DESTINATION_PATH} -n "${KUBE_NAMESPACE}"
2022/11/03 09:08:25 resource /builds/clients/***/***/Configurations/interpolated-files/api-gateway.deployment.yml: error converting YAML to JSON: yaml: line 50: mapping values are not allowed in this context
```

It seems the problem is about the api-gateway's interpolated file.
This is the first time I get this error and the problem doesn't occur when deploying in different environments.

The deploy pipeline we use is: 
```
  - project: 'platform/pipelines-templates'
    file: '/deploy/deploy-job-v3.yml'
    ref: 'master'
```

**Comment by davidebianchi** [comment, marked as answer]
Hi @theliuk! From the file you shared, I see the docker image is without the version but with a colon, so the yaml is invalid:

```yaml
image: nexus.mia-platform.eu/core/api-gateway:
```

Have you set the version with a public variable set empty?

**Reply by theliuk** [reply]
Yes, 
I didn't catch the `:` at the end of the line.

Thank you @davidebianchi
